docs/articles_en/learn-openvino/llm_inference_guide/llm-inference-hf.rst

```diff
@@ -235,17 +235,25 @@ includes **Dynamic quantization** of activations of 4/8-bit quantized MatMuls an
   insignificant deviation in generation accuracy.  Quantization is performed in a group-wise
   manner, with configurable group size. It means that values in a group share quantization
   parameters. Larger group sizes lead to faster inference but lower accuracy. Recommended
-  group size values are ``32``, ``64``, or ``128``. To enable Dynamic quantization, use
-  the corresponding inference property as follows:
+  group size values are ``32``, ``64``, or ``128``. Dynamic quantization is enabled **by default**
+  on the CPU device. To disable dynamic quantization, change execution mode from the ``PERFORMANCE mode``
+  to the ``ACCURACY mode``. More information on both modes are available in the
+  :ref:`Precision Control Guide <execution-mode>`
 
+  To change a group size value (e.g. to ``64``), you need to execute the following code:
 
   .. code-block:: python
 
      model = OVModelForCausalLM.from_pretrained(
          model_path,
-         ov_config={"DYNAMIC_QUANTIZATION_GROUP_SIZE": "32", "PERFORMANCE_HINT": "LATENCY"}
+         ov_config={"DYNAMIC_QUANTIZATION_GROUP_SIZE": "64", "PERFORMANCE_HINT": "LATENCY"}
```

Comment:
```suggestion
         ov_config={"DYNAMIC_QUANTIZATION_GROUP_SIZE": "64"}
```
Not changed in this PR, but if we're updating this doc, might as well remove the latency part, because it is not needed anymore (this is the default now always for OVModelForCausalLM)